azure-templates/steps-post-build.yml

```diff
@@ -79,10 +78,13 @@ steps:
     inputs:
       targetType: 'inline'
       script: |
-        New-Item `
-          -Path "$(archivePath)\$(Build.BuildNumber)\$(lvVersion)\$(architecture)" `
-          -ItemType 'Directory'
+        If (-not(Test-Path "$(archivePath)\$(Build.BuildNumber)\$(lvVersion)\$(architecture)"))
+        {
+          New-Item `
+            -Path "$(archivePath)\$(Build.BuildNumber)\$(lvVersion)\$(architecture)" `
+            -ItemType 'Directory'
+        }
         Copy-Item `
           -Path "$(buildOutputPath)\*" `
           -Destination "$(archivePath)\$(Build.BuildNumber)\$(lvVersion)\$(architecture)\" `
-          -Recurse
+          -Recurse   
```

Comment:
Nit: might be worth setting your editor to strip trailing whitespace on save.

Reply:
reinstalled VSCode and must have forgotten to reset that